docs/package.json

```
@@ -0,0 +1,17 @@
+{
+  "engineStrict": true,
+  "engines": {
+    "node": ">=10.6.0",
+    "npm": "NO LONGER USED - Please use yarn"
+  },
+  "devDependencies": {
+    "vuepress": "^1.0.2"
+  },
+  "scripts": {
+    "generate-docs": "cd .. && ./gradlew :velocidi-sdk:dokka",
+    "import-docs": "cp -R ../velocidi-sdk/build/docs/velocidi-sdk/ temp && cp -R .vuepress temp",
+    "dev": "yarn run generate-docs && yarn run import-docs && vuepress dev temp",
+    "clean": "rm -R temp && rm -R ../velocidi-sdk/build/docs/",
```

As discussed offline:

```suggestion
    "clean": "rm -Rf temp && rm -Rf ../velocidi-sdk/build/docs/",
```

---

Done in 5b3bbc8